.gitlab/ci/.gitlab-ci.global.yml

```diff
@@ -29,32 +29,32 @@
 
 .create-id-set:
   - section_start "Create ID Set" --collapsed
-  - demisto-sdk create-id-set -o ./Tests/id_set.json >> $ARTIFACTS_FOLDER/logs/create_id_set.log
-  - cp ./Tests/id_set.json $ARTIFACTS_FOLDER
+  - demisto-sdk create-id-set -o ./Tests/id_set.json >> "${ARTIFACTS_FOLDER}/logs/create_id_set.log"
```

Comment:
most of the changes here is adding quotes to `${ARTIFACTS_FOLDER}` -> `"${ARTIFACTS_FOLDER}"`